prism/src/main/java/org/apache/falcon/resource/proxy/ExtensionManagerProxy.java

```diff
@@ -460,15 +473,9 @@ protected APIResult doExecute(String colo) throws FalconException {
                 }else{
```

Comment:
in case there are partial updates the entities information is not stored and can't be deleted.